docs/source/updating_capabilities.md

```diff
@@ -256,15 +254,12 @@ You will need to export the following variables:
   to reset this variable for every channel you update.
 * `CORE_PEER_LOCALMSPID`: the MSP ID of the organization proposing the channel
   update. This will be the MSP of your peer organization.
-* `CORE_PEER_TLS_ROOTCERT_FILE`: the absolute path to the TLS cert of the peer
-  you will be using.
+* `TLS_ROOT_CA`: the absolute path to the TLS cert of your peer organization.
```

Comment:
In all the upgrade docs, TLS_ROOT_CA is always used to connect to the ordering node. Therefor, it is the TLS cert of the `ordering` nodes, not the `peer` nodes. It is better to rename it to ORDERER_TLS_ROOTCA for clarification.